ops/generate-subgraph.js

```diff
@@ -118,7 +118,7 @@ function combineFragments(fragments, isTemplate, addresses, missingAddresses) {
     } : {
       address: contractAddress,
       abi,
-      startBlock: startBlock
+      startBlock
```

Comment:
are you sure about that ?
how does it worked before ?

Reply:
This is just a small tweak to make the code nicer, the actual fix is the `parseInt` in the settings file